docs/apm/traces/working-with-tracing-data/tracing-dashboards.md

```diff
@@ -1,44 +1,41 @@
+Services must be active within the last 15 minutes to appear on the list. Recent inactivity will result in a grayed-out list entry.
 
 Each dashboard is a fully customizable set of panels based on automatically generated metrics using tracing data as input. You can get insight into microservice health by looking at stats from:
 
 * **Latency**: (real-time average) the average time it takes for entry spans for that service to complete. Apart from average you can also select p99, p95, p90, p50 options in service_latency_type filter to view other percentile aggregations.<br/> ![service-latency.png](/img/traces/service-latency.png)
 * **Requests**: (real-time counter) the number of entry spans reported by the service
 * **Errors**: (real-time counter) the number of entry spans for the service that finished with an error
 
-By clicking on any data point on the chart, just like for any other Sumo Logic dashboard, you can view the side panel's **Infrastructure** tab to drill down to related metrics or traces for the selected service.
+By clicking on any data point on the chart, you can view the side panel's **Entities** tab and drill down to related metrics or traces for the selected service.
+
+![Explore coffee app to drill down on infrastructure tab.png](/img/traces/coffee-app-infrastructure-tab.png)
 
 :::note
-Drilling to traces by application and to logs by both service and application are coming soon and are not working yet.
+Tracing metrics retention has [default metrics retention](/docs/apm/traces/working-with-tracing-data/view-and-investigate-traces/#time-range).
 :::
 
-![Explore coffee app to drill down on infrastructure tab.png](/img/traces/coffee-app-infrastructure-tab.png)
-
 ## Installing the Tracing App (Optional)
 
-The **Tracing - Application Services Health** app is automatically installed for all users of your organization once Sumo Logic detects OT-compatible tracing data coming from your instrumented services. The content is placed in **Sumo Logic Tracing - default dashboards** inside the **Admin Recommended** folder and is automatically available for all users in the organization.
+The **Tracing - Application Services Health** app is automatically installed for all users of your organization once Sumo Logic detects OpenTelemetry-compatible tracing data coming from your instrumented services. The content is placed in **Sumo Logic Tracing - default dashboards** inside the **Admin Recommended** folder and is automatically available for all users in the organization.
 
 :::note
-Do not modify or delete content in the **Admin Recommended** folder as it is maintained and updated automatically. If for any reason this gets removed, you can install the App manually from App Catalog. 
+Do not modify or delete content in the **Admin Recommended** folder as it is maintained and updated automatically. If for any reason this gets removed, you can install the App manually from App Catalog.
 :::
 
 To install the app, do the following:
```

Comment:
SS in line 43 needs to updated as well. @kimsauce SS is in : https://sumologic.slack.com/archives/C04BC5J7Q20/p1668635033883979